### DISCUSSION TITLE
Simple Commerce Starter Kit CSS

### DISCUSSION THREAD

**Discussion by joybcode** [duncanmcclean/simple-commerce#1135] [Help] [answered]
Hi,

How can we easily remove simple-commerce css styling/theme and replace it with our own styling?

**Comment by duncanmcclean** [comment]
Is there anything patricular you want to change?

**Reply by joybcode** [reply]
fonts, colours, button styles and general layout.

**Reply by joybcode** [reply]
We want to use styling from https://tailwindui.com/components#product-ecommerce

**Reply by duncanmcclean** [reply, marked as answer]
You can make changes to the Tailwind config and the Antlers templates as needed, Simple Commerce doesn't force you to keep the same look and style, in fact it's encouraged you change it.

**Reply by joybcode** [reply]
Thanks
